README.md

```diff
@@ -155,7 +155,7 @@ See [permissions](#permissions) for more details on the required GitHub token pe
 | --- | --- | --- |
```

Comment:
Do not forget to update the screenshot in the `README.md`

Reply:
Screenshot updated!

![Screenshot](https://raw.githubusercontent.com/jstmlr/commisery-action/improvement/bump_error/resources/example.png)